lib/schema/tablebuilder.js

```diff
@@ -46,15 +46,29 @@ class TableBuilder {
   }
 
   // The "timestamps" call is really just sets the `created_at` and `updated_at` columns.
-  timestamps() {
-    const method = arguments[0] === true ? 'timestamp' : 'datetime';
-    const createdAt = this[method]('created_at');
-    const updatedAt = this[method]('updated_at');
-    if (arguments[1] === true) {
-      const now = this.client.raw('CURRENT_TIMESTAMP');
-      createdAt.notNullable().defaultTo(now);
-      updatedAt.notNullable().defaultTo(now);
+
+  timestamps(param1, param2, useCamelCase) {
```

Comment:
Could you please rename `param1` and `param2` to how they are called in knex documentation, and also make `useCamelCase` a part of general params object, that would be the third function argument? We shouldn't keep adding positional parameters, so introducing a general param wrapping object would be helpful here.